### DISCUSSION TITLE
Some initial ideas

### DISCUSSION THREAD

**Discussion by stedy** [orcasound/orcamap-react#57] [Team Posts]
I made a fork of https://github.com/orcasound/orcamap-react and a PR with the goal of:
- Switching from google sheets to the [acartia API](https://acartia.io/api/v1/sightings/current)
- Updating/removing unnecessary node modules

There is a lot more to do on this map and some of the ideas I had for further improvement include:
- Clickable options to filter based on type of whale sighting
- More information in popup window for each sighting
- Scroll bar/selector to choose date range to show

I pulled together some of the map-related ideas from the [Orcasound Roadmap Trello board](https://trello.com/b/wBg0qhss/orcasound-roadmap)
- [Live map of ship traffic](https://trello.com/c/bypmwaRs/44-embed-a-live-map-of-ship-traffic)
- [Map of current listeners](https://trello.com/c/WscvCnsz/77-map-of-current-listeners)
- [Use a dynamic map](https://trello.com/c/Rncs0sI2/74-for-each-node-use-a-dynamic-map-not-a-static-one)

I am sure there are more ideas/suggestions out there, these were just the ones I was able to find and I think they could make some good starting points.

**Comment by scottveirs** [comment]
Thanks @stedy Zach! 

1. Were you able to get and use a token for Acartia API queries, or are you using the `current sightings` method, i.e.
```
GET https://acartia.io/api/v1/sightings/current
```

2. Re your further improvements, I got a request from an Orcasound member org for a monthly map by species, so you're on a valuable track. That said, an open question is whether orcamap-react should focus in first on SRKW visualization only (e.g. taking just SRKW points from Acartia and using them to create tracks, break down movement by pod, and/or project movements with forecast models). I do have whalemap.net reserved for more general visualizations (all cetacean species) but might suggest we start with SRKW challanges -- the current plan for the next hackathon in April... You may get some initial UI ideas from the [whaleaware.org project](https://whaleaware.org) down in CA.

3. The map related ideas you pulled from the Roadmap were mostly intended for the Orcasound's live-listening web app, specifically version 3 that for the first time has a dynamic map. The prototype is under development here [v3.orcasound.net](https://v3.orcasound.net).

**Reply by stedy** [reply]
Yes, I am just using that `current sightings` method as you mentioned. And thanks for those other links!
